### ISSUE TITLE
Auto-generated comment

### ISSUE THREAD

**Issue by zigomir** [SweetIQ/schemats#10]
Hey! Thanks for this cool project!

I'm guessing all the generated `.ts` files shouldn't be edited by user in most cases and it might make sense to add a top most comment about it. Something like:

```ts
/**
 * AUTO-GENERATED FILE - DO NOT EDIT!
 *
 * This file was generated with schemats node package:
 * $ schemats generate -c postgresql://user@localhost:5432/db -all used args-
 * 
 * Re-run command above if your DB schema was changed.
 * 
 */
export namespace novatus {
    // ...generated interfaces and types ...
}
```

what do you think?

**Comment by xiamx** [comment]
Yes good idea indeed. It will need some parsing to mask out database credentials. Pull-requests are welcomed. 👍

**Comment by xiamx** [comment]
Thanks @zigomir for working on this enhancement. The pull request has been merged and the changes are included in release 0.3.0

**Comment by zigomir** [comment]
No problem, thanks for guidance/feedback, and release!
